BPMRFFEApp/Db/bpmrffe.db

```diff
@@ -21,6 +21,30 @@ record(ai, "$(P)$(R)RFFEAtt-RB") {
     field(EGU, "dB")
 }
 
+# Temperature measurements
+
+record(ai, "$(P)$(R)RFFETempAC-Mon") {
+    field(DESC, "Measure temperature from AC board")
+    field(DTYP, "stream")
+    field(PINI, "YES")
+    field(SCAN, "5 second")
+    field(MDEL, "-1")
+    field(ADEL, "-1")
+    field(INP, "@bpmrffe.proto inMeasTempAC $(PORT)")
+    field(EGU, "oC")
+}
+
+record(ai, "$(P)$(R)RFFETempBD-Mon") {
+    field(DESC, "Measure temperature from BD board")
+    field(DTYP, "stream")
+    field(PINI, "YES")
+    field(SCAN, "5 second")
+    field(MDEL, "-1")
+    field(ADEL, "-1")
+    field(INP, "@bpmrffe.proto inMeasTempBD $(PORT)")
```

Comment:
I was thinking about this. I think to avoid code duplication, we should have an `inMeasTemp` function that takes an argument which specifies the board we are querying.

Reply:
Good idea. Thanks!

I'll implement this.